### DISCUSSION TITLE
How to migrate updated tables only

### DISCUSSION THREAD

**Discussion by alenvuletic** [f1db/f1db#41] [Q&A] [answered]
I use [Supabase](https://supabase.com/) to host PostgreSQL database for small app. I'm building using F1DB.

It took me ~2:30h to execute .sql via `psql` to see my database on Supabase servers (116MB and 226,081 lines to execute). As you can see, that's very time consuming, especially if I plan on doing it every racing weekend.

Is there a way to update only parts of the database that changed after racing weekend?

Btw. great project, one of a kind!

**Comment by marceloverdijk** [comment, marked as answer]
Hi @alenvuletic thx for reaching out.
There is no support for incremental updates.

I have no experience with Supabase but I'm wondering why it takes ~2:30h...
Creating the sqlite database on my macbook takes not even 1m...

Could you try to import it on your dev machine with a PostgreSQL instance running on it?
That should be fast as well. If that is fast it might be related how Supabase does the import.

**Comment by alenvuletic** [comment]
@marceloverdijk Thank you for quick response!

Yes you were right, I tried it on my local machine and it was imported in 15 sec. I'll move to Vercel probably.

Thanks!

**Reply by marceloverdijk** [reply]
Or cloudflare ;-)

🚣 Executed 224967 queries in 153.45 seconds (219 rows read, 2297443 rows written)

**Reply by alenvuletic** [reply]
I might try, I just tried vercel with psql and it's same speed.

**Reply by alenvuletic** [reply]
You did that migration with Argo tunnel right?

**Reply by marceloverdijk** [reply]
No, just using Wrangler from terminal.
